### PR TITLE
Add always remove listener rule

### PR DESCRIPTION
**Pull request by incendial**
### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix
[X] New rule
[ ] Changes an existing rule
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

### CODE REVIEW

File: lib/src/obsoleted/rules/always_remove_listener_rule.dart [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+    source.unit?.visitChildren(_visitor);
+
+    return _visitor.missingInvocations
+        .map((invocation) => createIssue(
+              rule: this,
+              location: nodeLocation(
+                node: invocation,
+                source: source,
+                withCommentOrMetadata: true,
```
Comment (dkrutskikh) [inline lib/src/obsoleted/rules/always_remove_listener_rule.dart:41]: @incendial please remove this argument. (this argument useful for get function or method node location)

**Review by dkrutskikh — CHANGES_REQUESTED** [review]

File: lib/src/obsoleted/rules/always_remove_listener_rule.dart (hunk 1 above)
Comment (incendial) [reply lib/src/obsoleted/rules/always_remove_listener_rule.dart]: Done.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 ## Unreleased
 
+* Add static code diagnostic `always-remove-listener`.
```
Comment (dkrutskikh) [inline CHANGELOG.md:5]: @incendial two lines with `Add static code diagnostic`

Comment (incendial) [reply CHANGELOG.md]: Fixed.

**Review by dkrutskikh — APPROVED** [review]

**Pull request merged**